platform/darwin/src/MGLStyleLayer.mm.ejs

```diff
@@ -229,6 +230,19 @@ namespace mbgl {
 <% if (paintProperties.length) { -%>
 #pragma mark - Accessing the Paint Attributes
 
++ (NSArray *)transitionKeys
+{
+    NSMutableArray *transitionKeys = [NSMutableArray array];
+<% for (const property of paintProperties) { -%>
+    [transitionKeys addObject:@"<%- camelizeWithLeadingLowercase(property.name) %>"];
+<% if (property.original) { -%>
+    [transitionKeys addObject:@"<%- camelizeWithLeadingLowercase(originalPropertyName(property)) %>"];
+<% } -%>
+<% } -%>
+
+    return transitionKeys;
+}
+
```

Comment:
Yeah, per https://github.com/mapbox/mapbox-gl-native/pull/8225/files#r104082076, you could do like this: 

```objc
+ (NSArray *)transitionKeys
{    
    return @[
<% for ...
       @"<%- camelizeWithLeadingLowercase(property.name) %>",
<% } -%>
    ];
}
```